### PR TITLE
Dockerfile: add php-imap-extension to all versions

### PR DESCRIPTION
**Pull request by mraenker**
Hi @Hipska,

here is the PR with the php-imap-extension added to all Dockerfiles of all versions. Hope, it's ok like that.

Cheers,
Martin

### CODE REVIEW

File: 2.3/Dockerfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ RUN apt-get update \
 		mysqli \
 		soap \
 		zip \
+		imap \
```
Comment (Hipska) [inline 2.3/Dockerfile:24]: Did you test this if there aren't any more libraries needed for this to work?

File: 2.4/Dockerfile [hunk 2]
```diff
@@ -21,6 +21,7 @@ RUN apt-get update \
 		mysqli \
 		soap \
 		zip \
+		imap \
```
Comment (Hipska) [inline 2.4/Dockerfile:24]: Did you test this if there aren't any more libraries needed for this to work?

File: 2.5/Dockerfile [hunk 3]
```diff
@@ -21,6 +21,7 @@ RUN apt-get update \
 		mysqli \
 		soap \
 		zip \
+		imap \
```
Comment (Hipska) [inline 2.5/Dockerfile:24]: Did you test this if there aren't any more libraries needed for this to work?

**Review by Hipska — COMMENTED** [review]
I'm also not sure to install this by default on the community iTop editions, as there is no extension included by default that needs this. It will only add unused storage when it will not be used for most of the users.

**Comment by mraenker** [comment]
> I'm also not sure to install this by default on the community iTop editions, as there is no extension included by default that needs this. It will only add unused storage when it will not be used for most of the users.

That's a fair point, I guess. I'll close this PR and create a new, fresh one.

**Comment by Hipska** [comment]
You could rework it in the same PR as well 😉 a new one is fine as well..

**Pull request closed without merging**